README.md

```
@@ -1,61 +1,201 @@
+    * ```params.pp```: Contains all the parameters necessary for the main configuration and template.
+    * ```init.pp```: Contains the main script of the module.
+* Custom
+    * ```custom.pp```: Used to add custom configurations to the Puppet module.
+
+## General Configuration Steps
 
-3. Run the relevant profile on the **Puppet agent**.
-    1. Default profile:
+1. Preparing the Puppet Environment:
+
+        Before starting the configuration steps, Puppet environment should be created and the necessary modules, manifests, and scripts should be added:
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)